release-notes/releases.json

```diff
@@ -5,7 +5,7 @@
         "version-runtime-display": "2.1.5",
         "version-sdk-display": "2.1.403",
         "version-aspnetcoremodule": "8.2.1991.0",
-        "date": "2018-09-11",
+        "date": "2018-02-10",
```

Comment:
Did you mean `2018-10-02` (i.e. Oct 2nd, not Feb 10th)

Reply:
Oops forgot that US switches the day and month. Fixed it in the next commit :)